src/handbook/customer/marketing/webinars.md

```diff
@@ -53,6 +54,7 @@ The following are the steps to produce a montly webinar.
 * Publish webinar recording
 * Social media post with link to webinar recording
 * In Hubspot send follow-up email to the registration list created in Hubspot that includes a link to the recording on YouTube.
+* For those who wanted to be contacted, change Life Cycle Stage property to MQL 
```

Comment:
@Hasmin-AC can we somehow use a survey from Hubspot instead? That makes it easier